packages/patternfly-4/react-virtualized-extension/package-lock.json

```diff
@@ -0,0 +1,5779 @@
+{
```

Comment:
@jenny-s51 sorry if I missed this before, I'm not sure when it snuck back in, but you've still got a `package-lock.json` file in here you should delete. Make sure you're not using the `npm` CLI commands in a yarn project, this file comes from using `npm install` instead of `yarn install`. `yarn.lock` is the only lockfile we do want to keep.

Reply:
Sorry about that -- I'm not sure how this snuck back in either! I've been using `yarn add`.

Reply:
Might have happened in a rebase. Old changes have a way of sneaking back in during a complex rebase :)